code/socialDistribution/utility.py

```diff
@@ -31,22 +17,18 @@ def get_post_like_info(post, author):
 
     else:
         request_url = post.public_id.strip('/') + '/likes'
-        response = make_request('GET', request_url)
+        status_code, response_body = api_requests.get(request_url)
 
-        if response.status_code == 200:
-            try:
-                likes_list = response.json()
-            except JSONDecodeError:
-                return None, 0
+        if status_code == 200:
+            likes_list = response_body["items"]
```

Comment:
Just a minor comment. While I think it is unlikely that this will happen, we might get 200 Ok but also a JSONDecodeError while parsing the response body, which would lead to a TypeError since response_body will be None.

Reply:
That's a good point. Fixed now.